src/tests/ftest/nvme/enospace.py

```diff
@@ -100,9 +100,14 @@ def ior_bg_thread(self):
 
         # Define the job manager for the IOR command
         job_manager = get_job_manager(self, "Mpirun", ior_bg_cmd, mpi_type="mpich")
-        self.create_cont()
-        job_manager.job.dfs_cont.update(self.container.uuid)
+
+        # create container
+        container = TestContainer(self.pool, daos_command=DaosCommand(self.bin))
+        container.get_params(self)
+        container.create()
+
         env = ior_bg_cmd.get_default_env(str(job_manager))
+        job_manager.job.dfs_cont.update(container.uuid)
```

Comment:
```suggestion
        self.add_container(pool=self.pool)

        env = ior_bg_cmd.get_default_env(str(job_manager))
        job_manager.job.dfs_cont.update(self.container.uuid)
```
Can't we use add_container() (and not import `TestContainer`)?

Reply:
Yes if there is major review comment , I will change it but I do not want to spin again and wait for all EC NVMe tests to run which takes day. I think release/2.0 is not going to be live for that long so not sure if we want to waste time